models/articles.models.js

```diff
@@ -0,0 +1,23 @@
+const db = require('../db/connection');
+
+exports.fetchArticleById = (article_id) => {
+  if (isNaN(article_id)) {
+    return Promise.reject({
+      status: 400,
+      msg: `Article ID must be a number.`,
+    });
+  }
+
+  const queryString = `SELECT * FROM articles 
+    WHERE article_id = $1`;
+
+  return db.query(queryString, [article_id]).then(({ rows }) => {
+    if (rows.length === 0) {
+      return Promise.reject({
+        status: 400,
```

Comment:
If an article isn't found but the request seems reasonable we would want to use a different error code and also test for that error code.